web/ui/src/common/jellyTable.js

```diff
@@ -131,68 +131,59 @@
                             sortedItems,
                             tableEntries;
 
-                        if (allItems === undefined) {
-
-                            // if just intitalized, show loading and default to empty array
-                            allItems = [];
-                            toggleNoData(false);
-                            return;
-
-                        } else if (angular.isObject(allItems) && !angular.isArray(allItems)) {
-
-                            // single-entry tables that are not arrays pass through once (eg config files)
+                        if (angular.isObject(allItems) && !angular.isArray(allItems)) {
+                            // single-entry tables that are not arrays and not null
                             tableEntries = utils.mapToArr(allItems);
                             $scope[tableID].loading = false;
                             toggleNoData(false);
-                            return;
-
                         } else if (allItems === null) {
-
-                            // if no data, remove loading and show no data message
```

Comment:
leave this comment, or put a similar comment. this code is hit when the backend explicitly sets a value to `null`, which indicates that the request completed successfully but there are no results (as opposed to data being `undefined`, meaning the request has not yet completed).